stdlib/2and3/parser.pyi

```diff
@@ -0,0 +1,33 @@
+from typing import Any, List, Sequence, Tuple, Union
+from types import CodeType
+
+import sys
+
+if sys.version_info >= (3, 6):
+    from builtins import _PathLike
+    _Path = Union[str, bytes, _PathLike]
+else:
+    _Path = Union[str, bytes]
+
+if sys.version_info < (3,):
+    def expr(source: Union[str, bytes]) -> STType: ...
+    def suite(source: Union[str, bytes]) -> STType: ...
+else:
+    def expr(source: str) -> STType: ...
+    def suite(source: str) -> STType: ...
```

Comment:
I Python 2, `str` and `bytes` are aliases. You probably meant to use `unicode`. But there is an easier way to write the above, without needing two branches by using `Text`, which is equivalent to `str` in Python 3 and `Union[str, unicode]` in Python 2 arguments.

Reply:
Ah, alright, I'll change it. I was aware of `Text`. but thought is was *only* unicode in py2, not a union of unicode and str.

Reply:
That's generally true, but in arguments it accepts both. It's a bit hidden in PEP 484:

> (Talking about Python 2)  For parameters typed as Text, arguments of type str as well as unicode should be acceptable.